Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -4593,10 +4593,20 @@ private List<string> GetWhereClauses(InternalItemsQuery query, IStatement statem
 
             if (query.ExcludeInheritedTags.Length > 0)
             {
-                var tagValues = query.ExcludeInheritedTags.Select(i => "'" + GetCleanValue(i) + "'");
-                var tagValuesList = string.Join(",", tagValues);
-
-                whereClauses.Add("((select CleanValue from itemvalues where ItemId=Guid and Type=6 and cleanvalue in (" + tagValuesList + ")) is null)");
+                var paramName = "@ExcludeInheritedTags";
+                if (statement == null)
+                {
+                    int index = 0;
+                    string excludedTags = string.Join(",", query.ExcludeInheritedTags.Select(t => paramName + index++));
+                    whereClauses.Add("((select CleanValue from itemvalues where ItemId=Guid and Type=6 and cleanvalue in (" + excludedTags + ")) is null)");
+                }
+                else
+                {
+                    for (int index = 0; index < query.ExcludeInheritedTags.Length; index++)
+                    {
+                        statement.TryBind(paramName + index, GetCleanValue(query.ExcludeInheritedTags[index]));
```

Comment:
Do we still need to "clean" the value if we're properly binding it now? What that cleaning does anyway?

Reply:
The "cleaning" basically calls String.Normalize() as well as converting the tag to lowercase.

I suspect it should be left in to keep the search case insensitive.